src/cobalt/media/decoder_buffer_allocator.cc

```diff
@@ -339,6 +339,7 @@ bool DecoderBufferAllocator::UpdateAllocationRecord(
     // TODO: Enable the following line once PrintAllocations() accepts max line
     // as a parameter.
     // reuse_allocator_->PrintAllocations();
+    return false;
```

Comment:
Why patching cobalt? Please revert

Reply:
Is this the core code of Cobalt? Then I agree. Where can we find the Device Porting layers in Cobalt?

Reply:
Everything is under _src/third_party/starboard/wpe/_

Reply:
Agree with the fact that the GST Buffers can be null. To cater for it is good. Adapting Cobalt sources is not good as we should only use the Device Porting Layers in cobalt (I saw wpe in the commit :-) )
But what would be the right action here if there is no GST buffer allocated?
@pawel-stanek-ml / @arun-madhavan-013

Reply:
The code in adaptation layer was fine. Returning and making sure nothing leaks is best one can do. I'd still expect things will go wrong from one one one way or another (broken playback in best case scenario, crashes in the worse one) since OOM condition got hit.